lblod/app-gelinkt-notuleren
# GN-5094: addition of migrations for the next legislation

### Overview
This PR is a big one. It contains the necessary migrations we need for the next legislation (2024-...) and the inauguration meeting.
Additionally, it also adds migrations to add the districts of Antwerp.
Specifically, the following migrations are included:
- Districts of Antwerp + mock users
- Fusiegemeenten 2025 + mock users
- Bestuursorganen (tijdspecialisaties) 2025
  * Gemeentes
  * OCMWs
  * Provincies
  * Note: Districts are already included in the seperate districts migrations
- Set-up of bestuursperiodes (concept taken from LMB) + connect them to the correct bestuursorganen (tijdspecialisaties)
  * Stored in public graph
  * Accessible through `/bestuursperiodes`
- Set-up of the kieslijsten of the local 2024 elections
  * Gemeentes
  * Districten
  * Provincies
  
TODOs:
- [x] Set-up correct mu-auth rules for: bestuursperiodes, kieslijsten, verkiezingen
- [ ] Fix the names of some 'fusiegemeenten', see https://binnenland.atlassian.net/browse/OP-3456

##### connected issues and PRs:
[GN-5094](https://binnenland.atlassian.net/browse/GN-5094)

### How to test/reproduce
- Start virtuoso
- Start the migrations service and ensure all migrations run successfully
- Start the rest of the app and open the frontend
- On the mock-login page, notice that you can now login as:
  * A district of Antwerp, for al list of districten see https://www.antwerpen.be/info/5603bc8bafa8a75dd48b45aa/districten
  * A fusiegemeente (which will be created in 2025)
- Create a meeting and ensure you can select both bestuursorganen from the current and next legislation

### Challenges/uncertainties
- These migrations are mostly adaptions from existing migrations from OP and LMB. I did have to make some adjustments:
  * Some errors in the predicates used
  * Some errors in the start-date of bestuursorganen
- The migrations added in this PR are quite extensive, I think these should encompass all data we need, but correct me if I'm wrong


### Checks PR readiness
